stemcell_builder/stages/bosh_aws_agent_settings/assets/agent.json

```diff
@@ -4,7 +4,9 @@
       "PartitionerType": "parted",
       "DevicePathResolutionType": "virtio",
       "CreatePartitionIfNoEphemeralDisk": true,
-      "ServiceManager": "systemd"
+      "ServiceManager": "systemd",
+      "DiskIDTransformPattern": "^(vol-(.+))$",
```

Comment:
Are two capture groups needed here? Doesn't seem like the outer capture group is used.

Reply:
that is correct. we can change this capture group